sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/Models/NetworkConfiguration.cs

```diff
@@ -33,13 +33,16 @@ public NetworkConfiguration()
         /// network subnet which the compute nodes of the pool will join. This
         /// is of the form
         /// /subscriptions/{subscription}/resourceGroups/{group}/providers/{provider}/virtualNetworks/{network}/subnets/{subnet}.</param>
+        /// <param name="dynamicVNetAssignmentScope">The scope of dynamic vnet
+        /// assignment.</param>
         /// <param name="endpointConfiguration">The configuration for endpoints
         /// on compute nodes in the Batch pool.</param>
         /// <param name="publicIPAddressConfiguration">The Public IPAddress
         /// configuration for Compute Nodes in the Batch Pool.</param>
-        public NetworkConfiguration(string subnetId = default(string), PoolEndpointConfiguration endpointConfiguration = default(PoolEndpointConfiguration), PublicIPAddressConfiguration publicIPAddressConfiguration = default(PublicIPAddressConfiguration))
+        public NetworkConfiguration(string subnetId = default(string), DynamicVNetAssignmentScope? dynamicVNetAssignmentScope = default(DynamicVNetAssignmentScope?), PoolEndpointConfiguration endpointConfiguration = default(PoolEndpointConfiguration), PublicIPAddressConfiguration publicIPAddressConfiguration = default(PublicIPAddressConfiguration))
```

Comment:
breaking change

Reply:
I didn't make this change, and I think it came from rebasing from main branch. Now it is same as main branch file

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/Models/NetworkConfiguration.cs#L42